lib/chef/provider/machine.rb

```diff
@@ -146,6 +146,12 @@ def machine_options(driver)
           result
         end
     }
+    
+    # I'm making the assumption here that you won't be using 2 different drivers
+    # in the same Ruby instance
```

Comment:
Please update the comment: the way I read it now (which led to my question), it sounds like there's some incompatibility that will happen if two drivers are loaded at the same time.